lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -736,14 +735,13 @@ def test_dates_tab_link_render(self, url_name, mfe_active):
 
         def assert_html_elements(assert_function, user):
             self.client.login(username=user.username, password=TEST_PASSWORD)
-            if mfe_active:
-                with override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True), \
-                     override_waffle_flag(COURSE_HOME_MICROFRONTEND_DATES_TAB, active=True):
-                    response = self.client.get(url, follow=True)
-            else:
+            with override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=mfe_active):
                 response = self.client.get(url, follow=True)
-            for html in html_elements:
-                assert_function(response, html)
+            if mfe_active and not user.is_staff:
+                assert 404 == response.status_code
```

Comment:
The rest of the code is just changing which waffles we look at right, but this looks like a behavior change?

Reply:
yup since we don't have date and outline flags anymore the legacy response would 404 when mfe is active

Reply:
OK - Carla set me straight here. My understanding of what this test was doing was bogus. I see it now and maybe your original pass at this before I commented was great. This version is also fine! Sorry - my confusion.